src/cpp/transport/UDPv4Transport.cpp

```diff
@@ -41,15 +41,15 @@ static void GetIP4s(std::vector<IPFinder::info_IP>& locNames, bool return_loopba
     locNames.erase(new_end, locNames.end());
 }
 
-static void GetIP4sUniqueInterfaces(std::vector<IPFinder::info_IP>& locNames, bool return_loopback = false)
-{
-    GetIP4s(locNames, return_loopback);
-    std::sort(locNames.begin(), locNames.end(),
-            [](const IPFinder::info_IP&  a, const IPFinder::info_IP& b) -> bool {return a.dev < b.dev;});
-    auto new_end = std::unique(locNames.begin(), locNames.end(),
-            [](const IPFinder::info_IP&  a, const IPFinder::info_IP& b) -> bool {return a.type != IPFinder::IP4_LOCAL && b.type != IPFinder::IP4_LOCAL && a.dev == b.dev;});
-    locNames.erase(new_end, locNames.end());
-}
+//static void GetIP4sUniqueInterfaces(std::vector<IPFinder::info_IP>& locNames, bool return_loopback = false)
```

Comment:
Remove if not used.

Reply:
Uncommented to use it again following the next comment.